INSTALL.md

```diff
@@ -1,5 +1,16 @@
 # Installing pandoc
 
+The simplest way to get the latest pandoc release is to use the graphical installer. Look out for the following name for your platform:
```

Comment:
Calling it a graphical installer isn't quite right - at least for the deb.  Binary installer?
Also, maybe break the line earlier; it seems rather long.